orchestra/migrations/0087_todo_task_data_migration.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2020-09-04 14:49
+
+from django.db import migrations
+from django.db.models import F
+from django.db.models import OuterRef
+from django.db.models import Subquery
+
+
+def fill_project_and_step_fields(apps, schema_editor):
```

Comment:
sanity-checking: does this code work, or is it still WIP?

Reply:
if it does work, how long does it take to run? :)

Reply:
It takes 41 seconds :)